dot/config.go

```diff
@@ -305,42 +312,47 @@ func KusamaConfig() *Config {
 				MutexProfileRate: kusama.DefaultPprofMutexRate,
 			},
 		},
-	}
+	}, nil
 }
 
 // PolkadotConfig returns a "polkadot" node configuration
-func PolkadotConfig() *Config {
+func PolkadotConfig() (*Config, error) {
+	defaultLogLevel, err := log.ParseLevel(westend.DefaultLvl)
```

Comment:
should we pass in polkadot instead of kusama here?

Reply:
done